components/apimgt/pom.xml

```diff
@@ -61,5 +61,6 @@
         <module>org.wso2.carbon.apimgt.common.gateway</module>
         <module>org.wso2.carbon.apimgt.common.jms</module>
         <module>org.wso2.carbon.apimgt.common.analytics</module>
+        <module>org.wso2.carbon.apimgt.eventing</module>
```

Comment:
We might have to build the eventing component at the beginning. Or else, dependency issues will occur.